### ISSUE TITLE
html tag lacks language attribute

### ISSUE THREAD

**Issue by garrett** [cockpit-project/cockpit#8587]
Every time an `<html>` tag is used, it should include a `lang` attribute.

For example:

```html
<html lang="en-US">
<html lang="de-DE">
<html lang="fr">
```

It takes the format of the 2 character language code, optionally followed by a dash and locale.

It's important for several reasons, and critical for accessibility. For a striking example of using no language code versus a language code, [watch this very short YouTube video](https://www.youtube.com/watch?v=ox5QVbZSPBk).

**Comment by garrett** [comment]
Addressed with respect to the shell in https://github.com/cockpit-project/cockpit/pull/8612/commits/15cf5d16d6c6f9dc03919007f722ca30dbffeef8 (PR #8612)

**Comment by garrett** [comment]
Additionally, **all** iframe-based documents _also_ need the proper `lang` to be added to their `<html>`, which is probably trickier than the previously-mentioned commit.
